src/app/modules/main/chat_section/module.nim

```diff
@@ -216,6 +216,17 @@ proc initContactRequestsModel(self: Module) =
 
   self.view.contactRequestsModel().addItems(contactsWhoAddedMe)
 
+method initListOfMyContacts*(self: Module) =
```

Comment:
@borismelnik since this method is called every time one opens create new group popup, I don't see that you are cleaning previously set items from the model, and you may have the same contact displayed multiple times if you open that popup multiple times.

And since this method will after the change clean previous items and populate a model again, maybe a better name for it `repopulateListOfMyContacts`.

Reply:
Just a small optimisation - i will clear the model when popup closed

Reply:
@saledjenic